schema/core.xsd

```diff
@@ -1195,7 +1195,7 @@
       </xs:sequence>
       <xs:attribute name="id" use="required" type="hv:ID" />
       <xs:attribute name="href" use="required" type="xs:string" />
-      <xs:attribute name="initial" type="xs:boolean" />
+      <xs:attribute name="selected" type="xs:boolean" />
```

Comment:
Intentional?

Reply:
Yes, this was a change we made in the navigation code